extensions/binding/org.eclipse.smarthome.binding.ntp/src/main/java/org/eclipse/smarthome/binding/ntp/handler/NtpHandler.java

```diff
@@ -96,8 +96,8 @@ public void initialize() {
 
 			Configuration config = getThing().getConfiguration();
 			this.hostname = (String) config.get(PROPERTY_NTP_SERVER);
-			this.refreshInterval = (BigDecimal) config.get(PROPERTY_REFRESH_INTERVAL);
-			this.refreshNtp = (BigDecimal) config.get(PROPERTY_REFRESH_NTP);
+			this.refreshInterval = new BigDecimal (config.get(PROPERTY_REFRESH_INTERVAL).toString());

```

Comment:
Yes, it is a workaround I think. I expect it to be a bigdecimal as the type in the thing-types.xml is integer.

and according to 
https://github.com/eclipse/smarthome/blob/master/bundles/config/org.eclipse.smarthome.config.core/src/main/java/org/eclipse/smarthome/config/core/Configuration.java#L83

this should be returned by the get as a BigDecimal. Actually even when running it step by step I dont quite understand why the error is happening.
Also I don't really see any changes that could have caused this new behaviour